addons/website/views/website_templates.xml

```diff
@@ -1239,7 +1239,7 @@
 <template id="option_header_off_canvas_logo_show" inherit_id="website.option_header_off_canvas" name="Show Logo (Off-Canvas)">
     <xpath expr="//a[hasclass('o_offcanvas_logo')]" position="replace">
         <a href="/" class="o_offcanvas_logo logo">
-            <span t-field="website.logo" t-options="{'widget': 'image', 'style': 'max-width: 200px'}" role="img" t-att-aria-label="'Logo of %s' % website.name" t-att-title="website.name" />
+            <span t-field="website.logo" t-options="{'widget': 'image', 'style': 'max-width: 200px', 'class': 'o_no_delete_button'}" role="img" t-att-aria-label="'Logo of %s' % website.name" t-att-title="website.name" />
```

Comment:
If you have to add a class on all t-field image to make them work... you should know this is not right. The bug is there before 13.5 but I guess we can backport the right fix. I think the right fix is to modify this code:
```javascript
        if (!this.isTargetParentEditable) {
            this.$el.add($customize).find('.oe_snippet_remove').addClass('d-none');
        }
```
to also add the d-none class if the target element is the inner content of a t-field which should not be removed (like the img tag in a t-field image). Make that fix in 13.5 and if it is safe enough I'll backport it maybe even before 13.4)

Reply:
Done :+1:  I couldn't find any other field than the image where we need to hide the remove button,  tell me if I forgot something ;)

Reply:
Not sure how you searched but just by grepping `'widget': 'image'` I have dozens of results. Go to a shop product and you have the same bug for instance :wink: